### DISCUSSION TITLE
Does TFT_eSPI 2.5.4 work with a LILYGO T-DISPLAY_S3 board ?

### DISCUSSION THREAD

**Discussion by cs0rfecs0rfe** [Bodmer/TFT_eSPI#3391] [Compatible displays and setup files]
Would you expect  TFT_eSPI 2.5.4 to work correctly on a LILYGO T-DISPLAY-S3 board please?

I have read conflicting advice:
This (https://github.com/teastainGit/LilyGO-T-display-S3-setup-and-examples) insists that a 'special' version of TFT_eSPI 2.5.0 from Lilygo is required (and indeed I had that working 3months ago):

The same author later said that  version 2.5.4 should work:
https://github.com/Xinyuan-LilyGO/T-Display-S3/issues/208#issuecomment-1853085293

Since a recent (automatic) update of something (due to espressif?) previously working code suddenly ran into compilation problems:

TFT_eSPI_ESP32_S3.c:96:8: error: 'gpio_get_level' was not declared in this scope; did you mean 'gpio_ll_get_level'?

and I attempted the workaround given in:
https://github.com/Bodmer/TFT_eSPI/issues/3355#issuecomment-2169396713

At this point I swapped to using the version of TFT-eSPI (2.5.4) that installs from the Arduino IDE library manager. I am aware of the need to update User_Setup_Select.h and have done so correctly. 

Whilst it all now compiles, none of my previously working sketches or the TFT_eSPI examples show anything on my boards' displays.

Would you expect this combination of board and library to work under the Arduino IDE with all current updates  to boards and libraries applied?

Thanks for any help/advice you can offer.

**Comment by SalihHortum** [comment]
I have the same problem with Lilygo-T-Display S3. 
Is there any working example and configuration for it?
